### ISSUE TITLE
Why isn't the -p flag on docker run working as expected?

### ISSUE THREAD

**Issue by tnithya** [projectcalico/calico-containers#791]
The FAQ has the following:
"Simply put, you don't need this flag with Calico and so Calico doesn't support it.
The -p flag tells Docker to set up port mapping to connect a port on the Docker host to a port on your container. This is useful because with Docker bridge networking, the container's IP address isn't reachable outside the host. But with Calico, the container's IP address is reachable not only within your cluster, but outside as well (see later questions for more detail).
If you're used to running your containers like this
docker run -d -p 8080:80 myhttpserver
and then accessing them via <docker-host-ip>:8080, you can instead run
docker run -d --publish-service myhttpserver.mynetwork.calico myhttpserver
and then access via <container-ip>:80. No more ephemeral ports, or port conflicts over which container gets to bind to port 80 (or any other port)!"

1. I think this should be updated since --publish-service is not a supported flag anymore (afaik)
2. If I want to use the docker IPVS tools like GORB, it is unable to see the ports that are exposed. Please advise on the way forward.
3. There is lot of difficulty when trying to access the containers from its host and more so from remote hosts. Is there any standard documentation available for enabling this? Am using RHEL 7 kernel 3.10.0-229.11.1.el7

**Comment by TrimBiggs** [comment]
Hi @tnithya,

There is a fix in (#832) for **(1)** in your list of concerns. I've updated the FAQ and there is now a new doc explaining how to configure port forwarding in Calico using linux commands [here](https://github.com/projectcalico/calico-containers/blob/master/docs/ExposePortsToInternet.md). Let me know if this is helpful to you.

Part **(2)** should also be covered by this fix. If you follow the instructions in the document I mention above, you should be able to see the exposed ports with the `iptables` command.

I am going to close the issue once this new Pull Request is merged, but feel free to reopen it if you are still having trouble with part **(3)** in your list.  If so, please let me know what you're seeing and what methods you've tried so I can help figure out a solution or workaround.

Thanks for bringing this issue to our attention!

**Comment by tomdee** [comment]
I'm closing since the PR that @TrimBiggs references has been merged
